packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -13,6 +13,7 @@ import uuid from 'uuid';
 
 import {InvalidUserCreation} from './convo-error';
 import ShareActivity from './share-activity';
+import ConversationConstant from './config';
```

Comment:
See my previous comment about importing these from a constants file.

Reply:
Okay, I i will see and replace with similar constant name here.